### PR TITLE
add missing button export_settings

### PR DESCRIPTION
**Pull request by MrTango**

### CODE REVIEW

**Comment by MrTango** [comment]
@sebastienbeau please review

**Review by bealdav — APPROVED** [review]

**Review by hparfr — APPROVED** [review]

File: connector_search_engine/views/se_index.xml [hunk 1]
```diff
@@ -200,6 +208,14 @@
                                                 icon="fa-upload"
                                                 type="object"
                                             >Export all binding</a>
+                                            <a
+                                                name="export_settings"
+                                                role="menuitem"
+                                                class="dropdown-item"
+                                                help="Force the export of all binding"
```
Comment (lmignon) [inline connector_search_engine/views/se_index.xml:215]: The help text is not the right one...

**Review by lmignon — CHANGES_REQUESTED** [review]
@MrTango Thank you for the contrib. A little comment...

File: connector_search_engine/views/se_index.xml (hunk 1 above)
Comment (simahawk) [inline connector_search_engine/views/se_index.xml:215]:
```suggestion
                                                help="Update or initialize index settings"
```

**Comment by bealdav** [comment]
Fixed by #188  cc @lmignon @simahawk

**Comment by lmignon** [comment]
Superseded by #188 where the last comment has been taken into account.

**Pull request closed without merging**